From 160deab5b96723955e207365f234bbb2d98cf1f9 Mon Sep 17 00:00:00 2001
From: Ran Wang <ran.wang_1@nxp.com>
Date: Wed, 13 Jun 2018 16:35:52 +0800
Subject: [PATCH 064/107] Revert "powerpc/T104xRDB: Fix endian access issue on EHCI intinalization"

There was an EHCI endian accessor issue. Now it's fixed by
commit 9829ce2ff25c ("usb: ehci: Fix accessors for big-endian platforms and descriptors")
Revert commit e6a727fffec7b2002e ("powerpc/T104xRDB: Fix endian access issue on EHCI intinalization")
due to it fix that issue in a wrong way. Actually, on T104x platform,
USB EHCI register use little endian.

Signed-off-by: Ran Wang <ran.wang_1@nxp.com>
---
 include/configs/T104xRDB.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/include/configs/T104xRDB.h b/include/configs/T104xRDB.h
index a054b97..6ef662b 100644
--- a/include/configs/T104xRDB.h
+++ b/include/configs/T104xRDB.h
@@ -624,7 +624,6 @@ $(SRCTREE)/board/freescale/t104xrdb/t1042d4_sd_rcw.cfg
 #ifdef CONFIG_USB_EHCI_HCD
 #define CONFIG_USB_EHCI_FSL
 #define CONFIG_EHCI_HCD_INIT_AFTER_RESET
-#define CONFIG_EHCI_DESC_BIG_ENDIAN
 #endif
 #endif
 
-- 
1.7.1

